src/lib/github/addIssuesToRepo.ts

```diff
@@ -11,21 +12,35 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
   const octokit = context.github;
   const { owner, repo } = getBasicRepoProps (context);
   const ghIssues = await getAllRepoIssues(context);
+
   newIssues.forEach(async ({title, body, authors}) => {
     const assignees = uniq(authors);
-    const currentGHIssue = find(ghIssue => ghIssue.title === title, ghIssues);
+
+    /**
+     * Am up to writing a function that removes checked out or no longer existing issue comments.
```

Comment:
Don't add block comments within function bodies. They should be at the top of a function definition. Use simple inline style for comments within function bodies.